components/input/dropdown/dropdown-item/DropdownItem.jsx

```diff
@@ -78,6 +80,7 @@ DropdownItem.defaultProps = {
 	variant: 'default',
 	dataAttrs: {},
 	selected: false,
+	customComponent: false,
```

Comment:
- Custom Component's default value should be null if that is a React Component. It cannot be false.